File: app/src/main/java/cat/teknos/pokedex/service/PokemonAPIService.java
package cat.teknos.pokedex.service;

import cat.teknos.pokedex.api.PokemonFetchResults;
import retrofit2.Call;
import retrofit2.http.GET;

public interface PokemonAPIService {

    @GET("pokemon?limit=100&offset=200")
    Call<PokemonFetchResults> getPokemons();
}

File: app/src/main/java/cat/teknos/pokedex/service/PokemonAPIService.kt
package cat.teknos.pokedex.service

import cat.teknos.pokedex.api.PokemonFetchResults
import retrofit2.Call
import retrofit2.http.GET

interface PokemonAPIService {
    @get:GET("pokemon?limit=151&offset=0")
    val pokemons: Call<PokemonFetchResults?>?
}